src/pages/login/__tests__/login.spec.ts

```diff
@@ -42,6 +49,8 @@ describe('LoginPage', () => {
         authenticationProvider = TestBed.get(AuthenticationProvider);
         appConfigProvider = TestBed.get(AppConfigProvider);
       });
+    store$ = TestBed.get(Store);
```

Comment:
As above, just to prevent default behaviour? Or is there a missing assertion?

Reply:
Same reason as above.

Reply:
Added it to remove complaints that store wasn't used (although it was required).